### PR TITLE
ensure users are created before databases

### PR DESCRIPTION
**Pull request by Morantron**
:raised_hands:

### CODE REVIEW

**Review by andresgutgon — APPROVED** [review]
Que facil :trollface:

**Comment by Morantron** [comment]
works on my machine :tm:

**Review by sauloperez — APPROVED** [review]

File: roles/database/tasks/main.yml [hunk 1]
```diff
@@ -8,17 +8,15 @@
...
-    tasks_from: users
   vars:
     postgresql_users:
       - name: "{{ database_user }}"
         role_attr_flags: "{{ database_role_attributes }}"
 
-- import_role:
+- include_role:
     name: geerlingguy.postgresql
-    tasks_from: databases
```
Comment (enricostano) [inline roles/database/tasks/main.yml:21]: If you remove this the whole role get executed, we just need `databases` tasks to be executed, same above for `users`.

**Review by enricostano — CHANGES_REQUESTED** [review]
This solution will execute the whole postgres role 3 times :scream:

**Comment by enricostano** [comment]
Dealt by professionals in https://github.com/coopdevs/timeoverflow-provisioning/pull/59 :trollface:

**Pull request closed without merging**